linux_sql/README.md

```diff
@@ -1,2 +1,51 @@
 # Linux Cluster Monitoring Agent
-This project is under development. Since this project follows the GitFlow, the final work will be merged to the master branch after Team Code Team.
+
+## Introduction
+Cluster Monitor Agent is a real-time internal tool that helps Linux Cluster Administration (LCA) team to collect and record the hardware specifications and node resource usages, such as CPU and memory information. The collected data is stored in a locally hosted PostgreSQL database and can be used for generating technical report. <br />
+Based on this generated report, better troubleshooting or resource planning would be performed by infrastructure team.
+
+## Architecture and Design
+The diagram below illustrates the architecture design of three nodes: <br /> <br /> 
+<img src="./assets/Design.png" width="550">
+
+- The `postgreSQL` instance is used to persist all the data <br />
+- The `Bash Scripts`, which is installed inside each node is consisted of two scripts: <br /> <br />
+    - `host_info.sh` collects the host hardware info and insert it into the database. It will be run only once at the installation time. <br />
+    - `host_usage.sh` collects the current host usage (CPU and Memory) and then insert into the database. It will be triggered by the `crontab` job every minute. <br />
+
+## Usage
```

Comment:
- Review README template and sample README file (missing content)
[https://www.notion.so/jarviscanada/README-md-bca70d9db5ea43edac12f4af1b4a0325#f146d34b24c542cf90665584a747f2cb](https://www.notion.so/jarviscanada/README-md-bca70d9db5ea43edac12f4af1b4a0325#f146d34b24c542cf90665584a747f2cb)